test/node/gruntTasks.js

```diff
@@ -41,6 +42,20 @@ var testBuild = function (index, callback) {
 
 };
 
+var veryfyIOswfFile = function () {
+    var originalFile = fs.readFileSync(srcPath + "aria/resources/handlers/IO.swf", "utf8");
```

Comment:
A binary file should probably not be read in `"utf8"` mode. I think it is better to remove the second parameter to have a buffer as a return value, and use the `equals` method to compare them.